osf/models/metaschema.py

```diff
@@ -195,14 +195,14 @@ class Meta:
         unique_together = ('schema', 'registration_response_key')
 
     schema = models.ForeignKey('RegistrationSchema', related_name='schema_blocks', on_delete=models.CASCADE)
-    help_text = models.TextField()
-    example_text = models.TextField(null=True)
+    help_text = models.TextField(null=True, blank=True)
+    example_text = models.TextField(null=True, blank=True)
     # Corresponds to a key in DraftRegistration.registration_responses dictionary
-    registration_response_key = models.CharField(max_length=255, db_index=True, null=True, blank=True)
+    registration_response_key = models.CharField(max_length=255, db_index=True, null=True, blank=True, default='')
```

Comment:
Didn't realize there were *two* migrations.  Please confirm that these changes are necessary and not just convenient. Presumably, we were making all relevant schema blocks before without this being an issue, and since (as mentioned in another comment), we start the process of updating schemablocks by deleting all previous blocks for the schema, we don't need to worry about coming back to set values later.